src/common/AndroidLauncher.ts

```diff
@@ -40,14 +40,16 @@ export class AndroidLauncher {
      * @param targetApp The bundle ID of the app to be launched.
      * @param appConfig An AndroidAppPreviewConfig object containing app configuration info.
      * @param serverPort The port for local dev server.
+     * @param targettingLwrServer Indicates whether we're previewing using LWC Dev Server (default behavior) or LWR Server.
      */
     public async launchPreview(
         compName: string,
         projectDir: string,
         appBundlePath: string | undefined,
         targetApp: string,
         appConfig: AndroidAppPreviewConfig | undefined,
-        serverPort: string
+        serverPort: string,
+        targettingLwrServer: boolean = false
```

Comment:
Nit: this should be `targetingLwrServer`.

Reply:
oops